contracts/interchainTokenService/InterchainTokenService.sol

```diff
@@ -49,15 +49,16 @@ contract InterchainTokenService is IInterchainTokenService, AxelarExecutable, Up
     bytes32 internal immutable chainName;
 
     bytes32 internal constant PREFIX_CUSTOM_TOKEN_ID = keccak256('its-custom-token-id');
-    bytes32 internal constant PREFIX_STANDARDIZED_TOKEN_ID = keccak256('its-cacnonical-token-id');
-    bytes32 internal constant PREFIX_STANDARDIZED_TOKEN_SALT = keccak256('its-cacnonical-token-salt');
+    bytes32 internal constant PREFIX_STANDARDIZED_TOKEN_ID = keccak256('its-standardized-token-id');
+    bytes32 internal constant PREFIX_STANDARDIZED_TOKEN_SALT = keccak256('its-standardized-token-salt');
 
     uint256 private constant SELECTOR_SEND_TOKEN = 1;
     uint256 private constant SELECTOR_SEND_TOKEN_WITH_DATA = 2;
     uint256 private constant SELECTOR_DEPLOY_TOKEN_MANAGER = 3;
     uint256 private constant SELECTOR_DEPLOY_AND_REGISTER_STANDARDIZED_TOKEN = 4;
 
-    // keccak256('interchain-token-service')-1
+    // TODO: For contract Id, we can stick to not using the -1 convention
```

Comment:
But why?

Reply:
forgot to add that we are doing it for our other contracts, so for consistency. Also, in this case the hash isn't used for a storage slot, so -1 benefit is a bit orthogonal